src/cdk/a11y/focus-monitor/focus-monitor.ts

```diff
@@ -99,10 +99,10 @@ export class FocusMonitor implements OnDestroy {
   private _windowFocused = false;
 
   /** The timeout id of the window focus timeout. */
-  private _windowFocusTimeoutId: number;
+  private _windowFocusTimeoutId: any;
```

Comment:
These are necessary, because the type now is `number | NodeJS.Timer`.

Reply:
Hmm. looks like these `any` casts are just needed for the legacy Saucelabs setup. Should we instead just declare the `global` type directly instead? I think that would be better than giving up on type safety. e.g.

```ts
declare const global: any;
```

Reply:
There's a comment at the top of that file about avoiding `declare const` so that's why I wrote it this way in the first place. Otherwise the same could've been written as `declare cost jest: any`.

Reply:
oh that's unfortunate. What is about `globalThis`? this should work as well and not need any Node types. Seems like it's made specifically for such platform-agnostic checks

Reply:
I avoided it since it isn't supported in some browsers (not sure if we care about them) and the workaround wasn't that much more code.

Reply:
okay, I don't feel very strongly about it. It seems supported by all of the browsers we make guarantees for (https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/globalThis), and it would save us from using `any` in all of `cdk/` (for the timers).